time.js

```diff
@@ -0,0 +1,39 @@
+document.addEventListener("DOMContentLoaded", function () {
+  let count = 0;
+  let timerInterval;
+  let timerDisplay = document.getElementById("timer");
+
+  function startTimer() {
+    if (!timerInterval) {
+      timerInterval = setInterval(() => {
+        count++;
+        timerDisplay.textContent = count;
+      }, 1000);
+    }
+  }
+
+  function pauseTimer() {
+    clearInterval(timerInterval);
+    timerInterval = undefined;
+  }
+
+  function resumeTimer() {
+    if (!timerInterval) {
+      startTimer();
+    }
+  }
+
+  function stopTimer() {
+    clearInterval(timerInterval);
+    timerInterval = undefined;
+    count = 0;
+    timerDisplay.textContent = count;
+  }
+
+  document.getElementById("start_button").addEventListener("click", startTimer);
+  document.getElementById("pause_button").addEventListener("click", pauseTimer);
+  document
+    .getElementById("resume_button")
+    .addEventListener("click", resumeTimer);
+  document.getElementById("stop_button").addEventListener("click", stopTimer);
```

Comment:
- Remove unnecessary breaking of lines. Place line breaks in proper places, and make sure it doesn't exceed 80 columns.
See: https://docs.google.com/presentation/d/1iCoz17mBxZtIQ9PifAW0jjKsuv4xDTlx-TPQxdLySow/edit#slide=id.g29e6ce31160_0_27